### ISSUE TITLE
[Tracking] Erroneous output in logs when generating a debug log from web interface

### ISSUE THREAD

**Issue by PromoFaux** [pi-hole/docker-pi-hole#1015]
Noticed this on a couple of different instances (interface name varies)

```
'unknown': unknown terminal type.
Device "eth0@if2" does not exist.
Device "eth0@if2" does not exist.
```

This does not happen when running `pihole -d` from inside the container on the command line

**Comment by rdwebdesign** [comment]
I think I found a solution for the first error `'unknown': unknown terminal type.`

**Comment by rdwebdesign** [comment]
`'unknown': unknown terminal type.` - Fixed by pi-hole/AdminLTE#2161

`Device "eth0@if2" does not exist.` - Fixed by https://github.com/pi-hole/pi-hole/pull/4664
